### DISCUSSION TITLE
Setting month start date.

### DISCUSSION THREAD

**Discussion by tmtribe20** [firefly-iii/firefly-iii#5228] [Firefly III]
Is it possible to have the month running from a set date such as from the 10th to the 9th, the reason I ask is I get paid on a certain date and I would like to have my budget running for a month from that date.

Thanks in advance

**Comment by JC5** [comment]
Nope.

https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/personal-finances/

**Reply by tmtribe20** [reply]
Cheers , now that I have read that I will use it that way cheers for prompt reply 👍🏼
